titus-ext/cassandra/src/main/java/com/netflix/titus/ext/cassandra/store/CassandraLoadBalancerStore.java

```diff
@@ -86,30 +106,43 @@
      */
     private final ConcurrentMap<String, SortedSet<JobLoadBalancer>> jobToAssociatedLoadBalancersMap;
 
-    private static final String GET_ALL_JOB_IDS = String
+    private static final String GET_ALL_ASSOCIATIONS = String
             .format("SELECT %s, %s, %s FROM %s;",
                     COLUMN_JOB_ID,
                     COLUMN_LOAD_BALANCER,
                     COLUMN_STATE,
-                    TABLE_LOAD_BALANCER);
-    private static final String INSERT_JOB_LOAD_BALANCER = String
+                    TABLE_LOAD_BALANCER_ASSOCIATIONS);
+    private static final String INSERT_ASSOCIATION = String
             .format("INSERT INTO %s(%s, %s, %s) VALUES (?, ?, ?);",
-                    TABLE_LOAD_BALANCER,
+                    TABLE_LOAD_BALANCER_ASSOCIATIONS,
                     COLUMN_JOB_ID,
                     COLUMN_LOAD_BALANCER,
                     COLUMN_STATE);
-    private static final String UPDATE_JOB_LOAD_BALANCER_STATE = String
-            .format("UPDATE %s SET %s = ? WHERE %s = ? AND %s = ?;",
-                    TABLE_LOAD_BALANCER,
-                    COLUMN_STATE,
-                    COLUMN_JOB_ID,
-                    COLUMN_LOAD_BALANCER);
-    private static final String DELETE_JOB_LOAD_BALANCER = String
+    private static final String DELETE_ASSOCIATION = String
             .format("DELETE FROM %s WHERE %s = ? AND %s = ?",
-                    TABLE_LOAD_BALANCER,
+                    TABLE_LOAD_BALANCER_ASSOCIATIONS,
                     COLUMN_JOB_ID,
                     COLUMN_LOAD_BALANCER);
 
+    private static Pair<JobLoadBalancer, JobLoadBalancer.State> buildLoadBalancerStatePairFromRow(Row row) {
+        String state = row.getString(COLUMN_STATE);
+        return Pair.of(
+                new JobLoadBalancer(row.getString(COLUMN_JOB_ID), row.getString(COLUMN_LOAD_BALANCER)),
+                state == null ? JobLoadBalancer.State.DISSOCIATED : JobLoadBalancer.State.valueOf(state.toUpperCase())
```

Comment:
No expected cases that I know, I just made the code a bit more defensive since it's being used in `Enum.valueOf`. I will add a warning log when it is `null`.